src/main/java/net/dv8tion/jda/api/entities/MessageEmbed.java

```diff
@@ -66,12 +66,18 @@
     public static final int VALUE_MAX_LENGTH = 1024;
 
     /**
-     * The maximum length the description and footer of an embed can have
+     * The maximum length the description of an embed can have
      *
-     * @see net.dv8tion.jda.api.EmbedBuilder#setFooter(String, String) EmbedBuilder.setFooter(text, iconUrl)
      * @see net.dv8tion.jda.api.EmbedBuilder#setDescription(CharSequence) EmbedBuilder.setDescription(text)
      */
-    public static final int TEXT_MAX_LENGTH = 2048;
+    public static final int TEXT_MAX_LENGTH = 4096;
```

Comment:
I feel like it would be better to make a DESCRIPTION_MAX_LENGTH and keep text as the footer maximum. That would also be less breaking

Reply:
Done